specification/apicenter/ApiCenter.Management/models.tsp

```diff
@@ -16,189 +16,214 @@ model OdataFilterParameter {
 }
 
 /** The expected types of origin */
-enum Origin {
+union Origin {
+  string,
+
   /** Origin with User */
-  user,
+  "user",
 
   /** Origin with System */
-  system,
+  "system",
 
   /** Origin with user and system */
-  `user,system`,
+  "user,system",
 }
 
 /** The type of action */
-enum ActionType {
+union ActionType {
+  string,
+
   /** an internal action */
-  Internal,
+  "Internal",
 }
 
 /** The provisioning state of the resource */
-enum ProvisioningState {
-  ...Azure.ResourceManager.ResourceProvisioningState,
+union ProvisioningState {
+  string,
+  Azure.ResourceManager.ResourceProvisioningState,
 }
```

Comment:
This is a union of `string` type and `enum` type, not union of `string` type and `string` literals.